### PR TITLE
03 oop

### PR DESCRIPTION
**Pull request by misvaleris**
Seems I've completed this task, @artemmoroz could you review it, please?

### CODE REVIEW

File: consoleApp/src/main/java/by/storeApp/StoreApp.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package by.storeApp;
```
Comment (artemmoroz) [inline consoleApp/src/main/java/by/storeApp/StoreApp.java]: if your goupId is by.issoft and artifactId is store - your module base package should be by.issoft.store.****. And please change it in every class/module.

File: consoleApp/src/main/java/by/storeApp/StoreApp.java [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+import by.store.RandomStorePopulator;
+import by.store.Store;
+
+import java.lang.reflect.InvocationTargetException;
+
+public class StoreApp {
+
+    public static void main(String[] args) throws InvocationTargetException, NoSuchMethodException, InstantiationException, IllegalAccessException {
+        RandomStorePopulator randomStorePopulator = new RandomStorePopulator();
+        Store store = randomStorePopulator.generateStore();
```
Comment (artemmoroz) [inline consoleApp/src/main/java/by/storeApp/StoreApp.java]: generating store from populator is not the best choice. please think one more time (may be relay on my drawings from prev call) and refactor that.

File: domain/src/main/java/by/goods/categories/Bike.java [hunk 3]
```diff
@@ -0,0 +1,10 @@
+package by.goods.categories;
+
+import by.goods.Category;
+
+public class Bike extends Category {
+
+    public Bike(String name) {
```
Comment (artemmoroz) [inline domain/src/main/java/by/goods/categories/Bike.java]: one question, could this category (Bike) has a name differrent from "Bike", if not), please think how you can simplify this constructor and category creation in populator class.

**Review by artemmoroz — CHANGES_REQUESTED** [review]

File: consoleApp/src/main/java/by/storeApp/StoreApp.java (hunk 1 above)
Comment (misvaleris) [reply consoleApp/src/main/java/by/storeApp/StoreApp.java]: updated

File: consoleApp/src/main/java/by/storeApp/StoreApp.java (hunk 2 above)
Comment (misvaleris) [reply consoleApp/src/main/java/by/storeApp/StoreApp.java]: updated

File: domain/src/main/java/by/goods/categories/Bike.java (hunk 3 above)
Comment (misvaleris) [reply domain/src/main/java/by/goods/categories/Bike.java]: updated

**Comment by misvaleris** [comment]
@artemmoroz I've updated code based on your comments, please review

**Review by artemmoroz — APPROVED** [review]

**Pull request merged**